examples/kubernetes/vtgate-service.yaml

```diff
@@ -1,9 +1,10 @@
 kind: Service
 apiVersion: v1
 metadata:
-  name: vtgate
+  name: vtgate-{{cell}}
```

Comment:
By our convention we should rename this to `-template` so people know they can't feed it directly to `kubectl`.